iba-cocktails.py

```diff
@@ -1,74 +1,89 @@
-service = Service(
-                executable_path=DRIVER_PATH, 
-            )
-driver = webdriver.Chrome(options=options, service=service)
-
-with open("cocktailURLs.csv") as f:
-    reader = csv.reader(f)
-    urlList = list(reader)
-cocktailList = []
-
-for val in urlList[0]:
-
-     driver.get(val)
-     
-     cocktailName = driver.find_element("xpath", '//*[@id="main-content"]/div/div/div/div/div[1]/div/div/h1').get_attribute('textContent')
-     
-     ingredients = driver.find_element("xpath", '/html/body/div[1]/div/div/div/div/div/div/div/div[2]/div[1]/p[1]').get_attribute('textContent').split('\n')
-     
-     def Convert(ingredients):
-        ingredientDick= {f'ingredient{i}': ingredients[i] for i in range(0, len(ingredients))}
-        return ingredientDick
-
-     method = driver.find_element("xpath", '//*[@id="main-content"]/div/div/div/div/div[2]/div[1]/p[2]').get_attribute('textContent')
-
-     garnish = driver.find_element("xpath", '//*[@id="main-content"]/div/div/div/div/div[2]/div[1]/p[3]').get_attribute('textContent')
-     
-     importance = driver.find_element("xpath", '/html/body/div[1]/div/div/div/div/div/div/div/div[1]/div/div/p/a[2]').get_attribute('textContent')
-     
-     cocktail = {
+#Load env variables
+def load_env():
+    load_dotenv()
+    return os.getenv("DB_STRING")
+
+#Connect to MongoDb using pymongo
+def connect_database(db_string):
+    client = pymongo.MongoClient(db_string)
+    db = client["cocktails-apd"]
+    return db["cocktails-iba"]
+
+#Set up Selenium and driver
+def setup_selenium(driver_path):
+    options = Options()
+    options.headless = True
+    options.add_argument("--window-size=1920,1200")
+    service = Service(executable_path=driver_path)
+    driver = webdriver.Chrome(options=options, service=service)
+    return driver
+
+#Get URLs from CSV and return a list of them
+def handle_url_csv(file_path):
+    with open(file_path) as f:
+        reader = csv.reader(f)
+        url_list = list(reader)[0]
+        return url_list
+
+#Get cocktail data from IBA website
+def get_cocktail_data(driver, url):
+    driver.get(url)
+
+    cocktailName = driver.find_element("xpath", '//*[@id="main-content"]/div/div/div/div/div[1]/div/div/h1').get_attribute('textContent')
+    ingredients = driver.find_element("xpath", '/html/body/div[1]/div/div/div/div/div/div/div/div[2]/div[1]/p[1]').get_attribute('textContent').split('\n')
+    method = driver.find_element("xpath", '//*[@id="main-content"]/div/div/div/div/div[2]/div[1]/p[2]').get_attribute('textContent')
+    garnish = driver.find_element("xpath", '//*[@id="main-content"]/div/div/div/div/div[2]/div[1]/p[3]').get_attribute('textContent')
+    importance = driver.find_element("xpath", '/html/body/div[1]/div/div/div/div/div/div/div/div[1]/div/div/p/a[2]').get_attribute('textContent')
+
+    def convert_to_dictionary(ingredients):
+        ingredient_dictionary = {f'ingredient{i}': ingredients[i] for i in range(0, len(ingredients))}
```

Comment:
Good work with the dictionary comprehension. You could even remove the variable and simply return this line. If you did that, though, may not make sense to have in its own func but should have a comment saying what you're doing above the code